features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1441,3 +1557,13 @@ CREATE INDEX IDX_CON_FILE_RES_ID ON IDN_CONFIG_FILE (RESOURCE_ID);
 
 -- SCOPE --
 CREATE INDEX API_ID_NAME_INDEX ON SCOPE (API_ID, NAME);
+
+-- XACML --
+CREATE INDEX IDX_POLICY_TENANT_ID ON IDN_XACML_POLICY (POLICY_ID, TENANT_ID);
```

Comment:
For any unique keys, lets add the unique key constraint in the table schema definition itself.

Reply:
Added for `IDN_XACML_POLICY` and `IDN_XACML_SUBSCRIBER` tables